boa_engine/src/context/intrinsics.rs

```diff
@@ -786,6 +794,8 @@ impl Default for IntrinsicObjects {
             is_nan: JsFunction::from_object_unchecked(JsObject::default()),
             parse_float: JsFunction::from_object_unchecked(JsObject::default()),
             parse_int: JsFunction::from_object_unchecked(JsObject::default()),
+            escape: JsFunction::from_object_unchecked(JsObject::default()),
+            unescape: JsFunction::from_object_unchecked(JsObject::default()),
```

Comment:
Shouldn't it have the `#[cfg(feature = "annex-b")]` annotation? Wouldn't it fail compilation if the `"annex-b"` feature is not enabled.